protobuf_definitions/message_formats.proto

```diff
@@ -858,4 +858,23 @@ message GuestPortCurrent {
   double gp2_bat = 2; // Current on GP2 battery voltage (A)
   double gp3_bat = 3; // Current on GP3 battery voltage (A)
   double gp_20v = 4; // Current on common 20V supply (A)
+}
+
+/**
+  * Vector with 3 elements
+  */
+message Vector3 {
+  double x = 1; // x-component
+  double y = 2; // y-component
+  double z = 3; // z-component
+}
+
+/**
+  * Imu data
+  */
+message Imu {
+  Vector3 accelerometer = 1; // Acceleration (g)
```

Comment:
We might want to normalize this vector at some point. We should communicate if this acc is relative to body or NED. Either in documentation or in the name, i.e. `accelerometer_body`, `accelerometer_ned`. I believe this is the acceleration in the body frame.